retrievalmarket/impl/requestvalidation/revalidator.go

```diff
@@ -220,12 +250,21 @@ func (pr *ProviderRevalidator) OnPullDataSent(chid datatransfer.ChannelID, addit
 		return true, nil, err
 	}
 
+	// Calculate how much data has been sent in total
 	channel.totalSent += additionalBytesSent
-	if channel.pricePerByte.IsZero() || channel.totalSent-channel.totalPaidFor < channel.interval {
+	if channel.pricePerByte.IsZero() || channel.totalSent < channel.interval {
```

Comment:
Can you explain the accounting change in the interval that led to this change of checking total sent against the interval rather than checking the total unpaid for bytes against the interval ?

Reply:
I added some detail in [this comment](https://github.com/filecoin-project/go-fil-markets/pull/540#discussion_r620131347)
Basically we're treating the interval as an absolute value instead of a delta now.